server/controllers/ical.coffee

```diff
@@ -27,7 +28,12 @@ createCalendar = (calendarName, callback) ->
         return callback err if err
 
         if events.length > 0
-            calendar.add event.toIcal() for event in events
+            for event in events
+                # first time app opening calendar and new calendar dummy event
+                endDateArray = moment(event.end, 'YYYYMMDD')._a
```

Comment:
What do you think of clarifying a little bit this part by doing:
```coffeescript
if moment(event.end, 'YYYYMMDD').year() > 1901
    calendar.add event.toIcal()
```
and to avoid comment __and__ clarify code:
```coffeescript
isDummyEvent = moment(event.end, 'YYYYMMDD').year() <= 1901
if not isDummyEvent
    calendar.add event.toIcal()
```
or ever define a isDummyEvent function in the Event class which could be reused later
```coffeescript
if not event.isDummy()
     calendar.add event.toIcal() # maximum clarity in my opinion
```